### PR TITLE
feature/expense-integration

### PR DESCRIPTION
**Pull request by ammarhaiderak**

### CODE REVIEW

File: src/store/models/transaction.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import { DatesInterface } from './root';
+import { User } from './user';
+
+
+export interface Transaction extends DatesInterface{
+    id?: number;
+    order?: null | number | Order;
+    payor?: number | User; 
+    payee?: number;
+    amount?: number | null;
```
Comment (OmerAli277) [inline src/store/models/transaction.ts]: Convert to string.

File: src/components/admin/ExpenseSummary.vue [hunk 2]
```diff
@@ -138,12 +109,90 @@
...
+        return this.user.company.credit;        
+
+      return 0;
+    },
+    startingBalance: function(): number {
+      
+      //first transaction where
+
+      return 0;
+    }
```
Comment (OmerAli277) [inline src/components/admin/ExpenseSummary.vue]: Remove this code

**Review by OmerAli277 — CHANGES_REQUESTED** [review]

**Review by OmerAli277 — DISMISSED** [review]

File: src/store/models/transaction.ts (hunk 1 above)
Comment (ammarhaiderak) [reply src/store/models/transaction.ts]: converted

File: src/components/admin/ExpenseSummary.vue (hunk 2 above)
Comment (ammarhaiderak) [reply src/components/admin/ExpenseSummary.vue]: done

**Review by OmerAli277 — APPROVED** [review]

**Review by mnouman11 — APPROVED** [review]

**Pull request merged**